website/content/api-docs/auth/cert.mdx

```diff
@@ -61,6 +61,19 @@ Sets a CA cert and associated parameters in a role name.
   string or array of `oid:value`. Expects the extension value to be some type
   of ASN1 encoded string. All conditions _must_ be met. Supports globbing on
   `value`.
+- `ocsp_enabled` `(bool: false)` - If enabled, validate certificates' revocation
+  status using OCSP.
+- `ocsp_ca_certificates` `(string: "")` Any additional CA certificates needed to
+  verify OCSP responses.  Provided as base64 encoded PEM data.
+- `ocsp_servers_override` `(array: [])`: A comma-separated list of OCSP server 
+  addresses.  If unset, the OCSP server is determined from the AuthorityInformationAccess 
+  extension on the certificate being inspected.
+- `ocsp_fail_open` `(bool: false)` - If true and an OCSP response cannot be fetched
+  or is of an unknown status, the login will proceed as if the certificate has not
+  been revoked.
+- `ocsp_query_all_servers` `(bool: false)` - If set to true, rather than accepting 
+  the first successful OCSP response, query all servers and consider the certificate 
+  valid only if all servers agree.
```

Comment:
Want add a note saying this would likely be the ideal configuration to enable when using Vault with separate PR clusters?